src/cmd/operator/manager.go

```diff
@@ -1,63 +1,102 @@
-/*
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package main
+package operator
 
 import (
-	"time"
-
 	"github.com/Dynatrace/dynatrace-operator/src/scheme"
-	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"github.com/pkg/errors"
 	"k8s.io/client-go/rest"
 	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/healthz"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 )
 
-func newManagerWithCertificates(ns string, cfg *rest.Config) (manager.Manager, func(), error) {
-	cleanUp := func() {}
-	mgr, err := ctrl.NewManager(cfg, ctrl.Options{
-		Namespace:          ns,
-		Scheme:             scheme.Scheme,
-		MetricsBindAddress: ":8383",
-		Port:               8443,
+const (
+	metricsBindAddress     = ":8080"
+	healthProbeBindAddress = ":10080"
+	operatorManagerPort    = 8383
+
+	leaderElectionId           = "dynatrace-operator-lock"
+	leaderElectionResourceLock = "configmaps"
+
+	livenessEndpointName = "/livez"
+	readyzEndpointName   = "readyz"
+	livezEndpointName    = "livez"
+)
+
+type managerProvider interface {
```

Comment:
why not use https://github.com/Dynatrace/dynatrace-operator/pull/876/files#diff-ac22401e6b044dcfe9a455ef86f78e3c149743333788c8912b83a6a4b1ef9cc6R8 instead ?

Reply:
cause that's a mocked provider, the link is a mocked manager